src/layout/FileUpload/AttachmentSummaryComponent2.tsx

```diff
@@ -28,6 +31,9 @@ export function AttachmentSummaryComponent2({ targetNode }: IAttachmentSummaryCo
 
   return (
     <AttachmentsMappedToFormDataProvider mappingTools={mappingTools}>
+      <Label weight={'regular'}>
```

Comment:
Bruk gjerne vår nye egne `<Label>`-komponent for å få det til å se likt ut som overalt ellers 😃

Reply:
Jeg bare gjorde det samme som allerede var gjort i Summary2: https://github.com/Altinn/app-frontend-react/blob/feature/subform/src/layout/Summary2/CommonSummaryComponents/SingleValueSummary.tsx#L33

Reply:
Den nye Label komponenten eksisterte ikke på det tidspunktet hvor summary2 ble satt opp. Det er vel den nye Label komponenten som gjelder fra nå av tenker jeg 😁